openhtf/output/callbacks/mfg_inspector.py

```diff
@@ -173,8 +173,8 @@ def _mangle_measurement(name, measured_value, measurement, mangled_parameters,
   We generate these by doing some name mangling, using some sane limits for
   very large multidimensional measurements.
   """
-  for coord, val in measured_value.value_dict.items(
-      )[:MAX_PARAMS_PER_MEASUREMENT]:
+  for coord, val in list(measured_value.value_dict.items(
```

Comment:
unnecessary

Reply:
I see this:
      for coord, val in measured_value.value_dict.items(
>         )[:MAX_PARAMS_PER_MEASUREMENT]:
E         TypeError: 'odict_items' object is not subscriptable

Reply:
Ah, I missed the indexing, sorry